app/utils/nonsipp/check/LoansCheckStatusUtils.scala

```diff
@@ -71,84 +69,9 @@ object LoansCheckStatusUtils {
     userAnswers: UserAnswers,
     srn: Srn,
     recordIndex: Max5000
-  ): Boolean = {
```

Comment:
Do we want to keep the existing logic as a fallback?
I'm thinking about the scenario that this change is rolled out to QA environment BEFORE ETMP changes are rolled out. I.e. prePopulation flag introduced and values we are passing in retained). If we remove this logic, there is a chance something will be broken in this transition period.

Reply:
I see... would we be able to detect if the changes are in ETMP? or would we use a feature flag maybe?

Reply:
We would need to remove the fallback logic later on. On or after 11th March -  this is when we expect these  ETMP changes to be rolled out.